x-pack/plugins/security_solution/public/timelines/components/timeline/body/renderers/host_name.tsx

```diff
@@ -41,10 +39,8 @@ const HostNameComponent: React.FC<Props> = ({
   const dispatch = useDispatch();
   const eventContext = useContext(StatefulEventContext);
   const hostName = `${value}`;
-
-  const { formatUrl } = useFormatUrl(SecurityPageName.hosts);
-  const isInTimelineContext = hostName && eventContext?.tabType && eventContext?.timelineID;
-
+  const isInTimelineContext =
+    hostName && eventContext?.enableHostDetailsFlyout && eventContext?.timelineID;
```

Comment:
To me I feel like we were intended not to open host details from alerts flyout as we check `tabTyp` here.
But given the ticket wants host details flyout to be able to open from alerts table but not alerts flyout, I think it would be more straight forward to use a specific flag to distinguish the two conditions.